node/src/main/kotlin/net/corda/node/services/keys/cryptoServices/AliasPrivateKey.kt

```
@@ -0,0 +1,10 @@
+package net.corda.node.services.keys.cryptoServices
+
+import java.security.PrivateKey
+
+/** [PrivateKey] wrapper to just store the alias of a private key. */
+class AliasPrivateKey(val alias: String): PrivateKey {
+    override fun getAlgorithm() = "ALIAS"
+    override fun getEncoded() = ByteArray(0)
```

Should we be throwing an exception here rather than returning an empty byte array? I can imagine callers would be confused by that.

---

I tried this, then if we ever store it to a Keystore it will break as it internally invokes this method. I will change it though as atm we don't store it to Keystores and if we did usually Keystores check for PKCS8 format.